plugins/SitesManager/templates/_siteWithoutDataTabs.twig

```diff
@@ -62,26 +62,33 @@
     </div>
 
     <div id="tracking-code" class="col s12">
-        {% if consentManagerName %}
+        {% if isNotificationsMerged %}
+            <p></p><p></p>
+            <div class="system notification notification-info merged-notification">
+                {{ notificationMergedMessage|raw }}
+            </div>
+        {% endif %}
+
+        {% if consentManagerName and isNotificationsMerged == false %}
         <p></p><p></p>
         <div class="system notification notification-info">
             <p> {{ 'PrivacyManager_ConsentManagerDetected'|translate(consentManagerName, '<a href="' ~ consentManagerUrl ~ '" target="_blank" rel="noreferrer noopener">', '</a')|raw }}
             </p>
             {% if consentManagerIsConnected %}
-                <p> {{ 'PrivacyManager_ConsentManagerConnected'|translate(consentManagerName)|raw }}
+                <p> {{ 'SitesManager_ConsentManagerConnected'|translate(consentManagerName)|raw }}
                 </p>
             {% endif %}
         </div>
         {% endif %}
 
-        {% if ga3Used %}
+        {% if ga3Used and isNotificationsMerged == false %}
         <p></p><p></p>
         <div class="system notification notification-info">
              {{ 'SitesManager_GADetected'|translate('Google Analytics 3', 'GA', '<a href="', 'https://matomo.org/faq/how-to/migrate-from-google-analytics-3-to-matomo/', '" target="_blank" rel="noreferrer noopener">', '</a>')|raw }}
         </div>
         {% endif %}
 
-        {% if ga4Used %}
+        {% if ga4Used and isNotificationsMerged == false %}
         <p></p><p></p>
         <div class="system notification notification-info">
             {{ 'SitesManager_GADetected'|translate('Google Analytics 4', 'GA', '<a href="', 'https://matomo.org/faq/how-to/migrate-from-google-analytics-4-to-matomo/', '" target="_blank" rel="noreferrer noopener">', '</a>')|raw }}
```

Comment:
Can we maybe move all the notification related stuff to PHP. It's easy to either set a "merged" notification or a single one in PHP and the twig code here simply displays what it gets without any further logic.

Reply:
@sgiehl moved to PHP